hail/python/hailtop/batch/backend.py

```diff
@@ -469,7 +470,7 @@ def _cp(src, dst):
                 resources['storage'] = job._storage
 
             j = bc_batch.create_job(image=job._image if job._image else default_image,
-                                    command=['/bin/bash', '-c', cmd],
+                                    command=[job._shell if job._shell else '/bin/bash', '-c', cmd],
```

Comment:
If you're concerned about duplicating bash, you can do something like `DEFAULT_SHELL = '/bin/bash'` in the Backend abstract base class.